server/settings.py

```diff
@@ -31,9 +31,10 @@ def env(variable, fallback_value=None):
 if init_data.exists():
     INIT_DATA_PATH = init_data
 
-INSTALLED_APPS.extend([
+INSTALLED_APPS = [
     'apps.languages',
-])
+    'anp.photo',
+]
```

Comment:
why replacing `extend` by `=`? Currently INSTALLED_APPS is empty in `superdesk-core` (https://github.com/superdesk/superdesk-core/blob/master/superdesk/default_settings.py#L362) but if in the future something is added there, it will be overridden here, and that can be hard to debug.

Reply:
the idea was to keep it empty in core, that's why `CORE_APPS` was added